apps/docs/src/lib/components/TopBar/TopBar.svelte

```diff
@@ -60,34 +62,61 @@
 		localStorage.setItem('recentSearches', JSON.stringify(existingSearches));
 	}
 
+	function changePreviousSection(newVal: string) {
+		previousSection = newVal;
+		return newVal;
+	}
+
+  function validateSearchLink(searchLinkSection: string) {
+    return searchLinkSection === previousSection
+  }
+
 	// @ts-nocheck
+
+	const useStyles = createStyles(() => ({
+		'.svelteui-Modal-inner > div': {
+			width: '100%',
+			display: 'block !important'
+		},
+		'.svelteui-Modal-inner': {
+			display: 'block !important'
+		},
+		'.svelteui-Modal-modal': {
+			width: '100%',
+			maxWidth: '40rem',
+			background: 'linear-gradient(135deg, #3e97e6, #86b8e3) !important',
```

Comment:
I actually kinda preferred like it was before, with the menu in the middle and white background, with the blue accent on hover. Why this change?
Also, one thing to improve here is focus on tab and arrow keys, right now (in the deployed version as well) the items  are not focusable with arrow keys. I'll add to the list in the issue

Reply:
I changed the background because I thought that it would provide some nice contrast in the white background. As to the arrow keys, I'm not really sure how to do it, so could you point me in a direction?

Reply:
Hum, probably with the `on:keydown` handler in the modal and saving a focused state. I would leave this arrow keys improvement for later, I think the one that should already be working is with the `Tab` key. The items seem like they are not focusable, so it's probably a bug. You don't need to do this in this PR though.

Reply:
Ok 👍

Reply:
Actually when I checked the tabs, it seemed to be working perfectly. The only problem is that there is no way for the user to know which element the focus is on currently

Reply:
Sorry, was away from the PC these days, will take a look later! Thank you for your help!

Reply:
👍